src/App.tsx

```diff
@@ -3,11 +3,19 @@
 import { Redirect, Route } from 'react-router-dom';
 import { IonApp, IonRouterOutlet, setupIonicReact } from '@ionic/react';
 import { IonReactRouter } from '@ionic/react-router';
+<<<<<<< HEAD
```

Comment:
It looks like maybe you didn't finish resolving a merge conflict. I don't think line 6,12, or 18 should be like that. There's a few more lines down below with the same issue